tests/main/snap-discard-ns/mount-py2.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python2
+"""
+Pure-python limited-use wrapper around the mount library function.
+"""
+from __future__ import print_function, absolute_import, unicode_literals
+
+from argparse import ArgumentParser
+from ctypes import CDLL, c_char_p, c_long, get_errno
+from ctypes.util import find_library
+from os import strerror
+from sys import stderr
+
+__all__ = ('mount', )
+
+
+class MountOpts(object):
+    """MountOpts contain various flags for the mount system call."""
+
+    Bind = 4096
+
+
+def mount(source, target, fstype, flags=0, data=""):
+    # type: (unicode, unicode, unicode, int, unicode) -> None
+    """mount is a thin wrapper around the mount library function."""
+    libc_name = find_library(b"c")
+    if libc_name is None:
```

Comment:
`if not libc_name:`?

Reply:
PEP8 suggests testing None the way I did to avoid masking the distinction between None and other things that evaluate to false in boolean context.